CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.2.1] - 2025-07-17
+
+### Added
+ - using THS  ^1.1.3" with out-of-bounds handling
```

Comment:
looks like there's an extra space after THS instead of a quote mark

Reply:
oh indeed, I'll get that Thanks @voj